src/js/frontend/src/components/entityData.js

```diff
@@ -0,0 +1,44 @@
+let d3 = require('d3');
+let api = require('./api');
+
+let entityData = {
+  fetch: function(entityType, entityId, onSuccess) {
+    let url = this.apiURL(entityType, entityId);
+    console.log("Fetching from " + url);
```

Comment:
I'd prefer to change to `console.debug`.  There won't be many of them, and they are useful (even for a production deployment).